lib/core/src/lib/form/components/widgets/core/form-field.model.ts

```diff
@@ -188,7 +190,8 @@ export class FormFieldModel extends FormWidgetModel {
             this.dynamicDateRangeSelection = json.dynamicDateRangeSelection;
             this.regexPattern = json.regexPattern;
             this.options = json.options || [];
-            this.hasEmptyValue = json.hasEmptyValue;
+            this.hasEmptyValue = json?.hasEmptyValue ?? this.hasEmptyOption(this.options);
+            this.emptyOption = this.hasEmptyValue ? this.getEmptyOption(this.options) : undefined;
```

Comment:
I will stick with that because when this.hasEmptyValue (calculated before) is false we can explicitly return undefined instead of calling getEmptyOption(this.options)

Reply:
Or I will switch to that:
```
this.emptyOption = this.getEmptyOption(this.options);
this.hasEmptyValue = json?.hasEmptyValue ?? !!this.emptyOption;
```